packages/api-derive/src/chain/util.ts

```diff
@@ -16,7 +16,7 @@ import { memo, unwrapBlockNumber } from '../util';
 // we would emit code with ../<somewhere>/src embedded in the *.d.ts files
 export type { BlockNumber } from '@polkadot/types/interfaces';
 
-export function createBlockNumberDerive <T extends { number: Compact<BlockNumber> | BlockNumber }> (fn: (api: DeriveApi) => Observable<T>): (instanceId: string, api: DeriveApi) => () => Observable<BlockNumber> {
+export function createBlockNumberDerive<T extends { number: Compact<BlockNumber> | BlockNumber }>(fn: (api: DeriveApi) => Observable<T>): (instanceId: string, api: DeriveApi) => () => Observable<BlockNumber> {
```

Comment:
Weird that CI is not complaining with this change. Some Eslint rule is not quite working.